tests/cli/test_cli.py

```diff
@@ -739,6 +739,14 @@ def test_upload1(ctx):
         2 + 1, _run_command([cl, 'cat', uuid])
     )  # Directory listing with 2 headers lines and one file
 
+    # Upload a file that exceeds the disk quota
+    username = _run_command([cl, 'uinfo', '-f', 'user_name'])
+    prev_dq = _run_command([cl, 'uinfo', username, '-f', 'disk']).split(' ')[2]
+    _run_command([cl, 'uedit', username, '--disk-quota', '2'])
+    # expect to fail when we upload something more than 2 bytes
+    _run_command([cl, 'upload', 'codalab.png'], expected_exit_code=1)
+    _run_command([cl, 'uedit', username, '--disk-quota', prev_dq])
```

Comment:
This command should be run regardless of whether this test fails or not. Could we maybe move this command to the "cleaning up" section of this file so that that is the behavior of it?

Reply:
how would it know the previous disk quota? Should I make it a global variable? Or just set it to the default of 100g

Reply:
You can run `cl uinfo` to get the previous disk quota

Reply:
if the test fails and it goes to cleanup, wouldn't the previous disk quota from `cl uinfo` be the wrong value? In this case, 2? Seeing as the default is 100g, I think we should just reset to that no matter what. Thoughts?

Reply:
Sorry if I wasn't clear. I meant that you can run `cl uinfo` in the `__enter__` function to get the original disk quota (which would run before the tests start in the first place), then restore the disk quota using `cl uedit` in `__exit__`. I think it's probably better to keep this general than hardcoding 100g in the code base here.